openerp/api.py

```diff
@@ -911,6 +920,7 @@ def __init__(self):
         self.todo = {}                  # recomputations {field: [records]}
         self.mode = False               # flag for draft/onchange
         self.recompute = True
+        self.recompute_list = []        # list of old api compute fields to recompute
```

Comment:
I don't like the suffix `_list`, it does not provide any useful information.
I suggest `recompute_old` instead, to make it clear that it is about the old-API recomputation stuff.